packages/react/src/patterns/sub-patterns/PictogramItem/__stories__/PictogramItem.stories.js

```diff
@@ -55,12 +56,25 @@ storiesOf('Patterns (Sub-Patterns)|PictogramItem', module)
           return Touch;
       }
     };
+
+    const selected = {
+      src: selectPictogram(pictogram),
+      ariaLabel: text('Aria-label:', ''),
+      ariaHidden: text('Aria-hidden:', ''),
```

Comment:
I don't think we want both at the same time. Just stick with ariaLabel, and add some placeholder text.